pkg/combustion/templates/script-base.sh.tpl

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
-# combustion: network
 set -euo pipefail
 
+{{ if .NetworkScript -}}
```

Comment:
Surprised we made it this long without needing to templatize the base combustion script.